### PR TITLE
Improve the README

### PR DESCRIPTION
**Pull request by RaptaG**
Not very serious changes, just some enhancements to make it look better!
Suggestions wanted!

Signed-off-by: RaptaG <77157639+RaptaG@users.noreply.github.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,44 +28,33 @@ Prebuilt Development builds are provided for **Linux**, **Windows** and **macOS*
...
+These packages are also availiable to all the distributions based on the ones mentioned above.
 
 ## Community & Support
 
-Feel free to create a GitHub issue if you find a bug or want to suggest a new feature. We have multiple community spaces where other community members can help you.
+Feel free to create a GitHub issue if you find a bug or want to suggest a new feature. We have multiple community spaces where other community members can help you:
 
-#### Join our Discord server:
-[![Prism Launcher Discord server](https://discordapp.com/api/guilds/1031648380885147709/widget.png?style=banner2)](https://discord.gg/prismlauncher)
+1) **Our Discord server:**
```
Comment (flowln) [inline README.md]: I think this could be a bullet list instead of a numbered one, since imo there's no hierarchy or order to those in principle.

**Review by flowln — COMMENTED** [review]
this looks mostly good to me, thanks!

File: README.md (hunk 1 above)
Comment (RaptaG) [reply README.md]: Done!

**Review by flowln — APPROVED** [review]
thanks!

**Review by DioEgizio — APPROVED** [review]

**Review by Scrumplex — APPROVED** [review]

**Comment by RaptaG** [comment]
🎉

**Pull request merged**